packages/new_stats/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:18 AS build
+
+WORKDIR /app
+
+COPY . /app/
+
+# Install yarn and build project
+RUN yarn install
+RUN yarn lerna run build --no-private && yarn workspace @threefold/newstats build
```

Comment:
same here docker build fails on this step 'error Unknown workspace "@threefold/newstats".'

Reply:
any package should be prefixed with `@threefold` in the sdk, So we should fix the package name